Remotion/ObjectBinding/Web/UI/Controls/BocRendererBase.cs

```diff
@@ -158,7 +158,16 @@ public virtual string CssClassReadOnly
     /// </remarks>
     public virtual string CssClassDisabled
     {
-      get { return "disabled"; }
+      get { return CssClassDefinition.Disabled; }
+    }
+
```

Comment:
Oh, there's also a pattern here, please update the BocRenderer implementations to use the to be newly introduced property `CssClassRemotionThemed`